engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oidc/OidcService.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.ui.security.oidc
+
+import pl.touk.nussknacker.ui.security.oauth2.{DefaultJwtAccessToken, DefaultOpenIdConnectAuthorizationData, JwtValidator, OAuth2ClientApi, OpenIdConnectService, OpenIdConnectUserInfo}
+import sttp.client.{NothingT, SttpBackend}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class OidcService(configuration: OidcAuthenticationConfiguration)
```

Comment:
what's the difference in terms of intended use between OidcService and OpenIdConnectService ?

Reply:
The OidcService is concrete - with case classes specified for JWT parsing.
The OpenIdConnectService is generic - you can pass your own type for JWT parsing. Maybe the name could be better here.

My aim generally is to have a generic implementation in the oauth2 package (in case someone needs to write pseudo-authentication with OAuth2 with own requirements) and a specific one in the oidc package

Reply:
Let's change name to sth more meaningful... Or with a comment...